### PR TITLE
Check the pixel mode for ORs is set to ORIG

### PR DESCRIPTION
**Pull request by jeanconn**
## Description
Check the PEA pixel mode for ORs is set to ORIG

## Testing

Tested out of ska3-matlab-2023.4rc6 though that should not be required.

### Unit tests

- [x] No unit tests

### Functional tests

I edited JUL0323A backstop to remove the first command to switch back to ORIG after perigee for this output:

https://icxc.cfa.harvard.edu/aspect/test_review_outputs/starcheck-pr414/jul0323a_orig_removed.html

I edited JUL0323A backstop to move the first command to switch back to ORIG after perigee to be in the next OR for this output:

https://icxc.cfa.harvard.edu/aspect/test_review_outputs/starcheck-pr414/jul0323a_orig_moved.html

### CODE REVIEW

File: starcheck/utils.py [hunk 1]
```diff
@@ -103,8 +105,74 @@ def get_data_dir():
...
+class OriginalPixelModeSelectTransition(FixedTransition):
+    command_attributes = {'tlmsid': 'AAC1CCSC', 'hex': '3492040'}
+    state_keys = ['pixel_mode']
+    transition_key = ['pixel_mode']
+    transition_val = ['ORIG']
+
+
+def get_pixel_mode_kadi_continuity(backstop_file):
+    rltt, sched_stop = get_rltt_and_sched_stop(backstop_file)
+    if rltt.date < '2023:178':
```
Comment (jeanconn) [inline starcheck/utils.py:124]: I didn't see a great way to add a default or date-based continuity for the new FixedTransitions I defined, so I just made a new method to hard-code it for this.

**Comment by taldcroft** [comment]
The new custom pixel state is great! 

I think there is a tidier path to getting the commands and subsequent states that reuses functionality built into kadi. See: https://gist.github.com/taldcroft/fb0998994cfebc22d06791dc5540c723

**Comment by taldcroft** [comment]
I would suggest to close this for now. If we get back to switching pixel mode again we can re-open it.

**Comment by javierggt** [comment]
**NOTE**: This PR was closed, because pixel mode is not going to change in the near future.

After backstop 7.9 (FSDS-83), there were some changes in backstop. Specifically:

- switch to pixel telemetry `ALT`
  ```
  - TLMSID= AAC1CCSC, HEX= 348E040, MSID= AAC1CCSC, AACKSUM=71 , AAWRDCNT=2 
  + TLMSID= AAC1PTLM, CMDS= 3, FIELD=2
  ```
- switch to pixel telemetry mode `ORIG`
  ```
  - TLMSID= AAC1CCSC, HEX= 3492040, MSID= AAC1CCSC, AACKSUM=73 , AAWRDCNT=2 
  + TLMSID= AAC1PTLM, CMDS= 3, FIELD=0
  ```

Similarly, if the background algorithm is set:
  - switch to legacy background algorithm
  ```
  - TLMSID= AAC1CCSC, HEX= 34B2040, MSID= AAC1CCSC, AACKSUM=89 , AAWRDCNT=2
  + TLMSID= AAC1BGD, CMDS= 3, FIELD=0
  ```
  - switch to dynamic background algorithm
  ```
  - TLMSID= AAC1CCSC, HEX= 34B0040, MSID= AAC1CCSC, AACKSUM=88 , AAWRDCNT=2
  + TLMSID= AAC1BGD, CMDS= 3, FIELD=1
  ```

**Pull request closed without merging**